.github/workflows/security.yaml

```diff
@@ -1,121 +1,86 @@
 name: "Security checks"
```

Comment:
Here is an example of the security check passing, correct? https://github.com/bojinovic/filecoin-solidity-library/actions/runs/9171124449